src/main.rs

```diff
@@ -203,6 +204,7 @@ fn main() -> Result<(), String> {
     }
 
     xcb::ungrab_pointer(&conn, xcb::CURRENT_TIME);
+    xcb::ungrab_key(&conn, 9, window, xcb::MOD_MASK_ANY as u16);
```

Comment:
Maybe this should be behind a function in `mod.rs` to at least preserve the grab/ungrab logic (and the magic number) in the same place?

Reply:
I think if we just make a `const ESC = 9` like you said that'd be fine. Might be a good idea to do it for all the other magic numbers we have